.circleci/config.yml

```diff
@@ -78,7 +78,7 @@ workflows:
       - test:
          name: "code-check"
          image: "python:3.6"
-         toxenv: "protocheck,mypy,black,flake8,flake8-docstrings"
+         toxenv: "protocheck,mypy,black,flake8,flake8-docstrings,verify-sweeps"
```

Comment:
This tox target makes sure that the sweeps version that was vendored to `wandb/sweeps` matches the contents of the https://github.com/wandb/sweeps git repository @ src/sweeps and that github.com/wandb/sweeps requirements.txt matches requirements.sweeps.txt in this repo